src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import './CommentInfo.scss';
+
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
```

Comment:
`<strong className="CommentInfo__name">{`${name} by `}</strong>`
What do you think?  Than we can remove str 23

Reply:
'by' shouldn`t be bold) (what the tag 'strong' does)